doc/classes/NavigationAgent2D.xml

```diff
@@ -137,6 +146,17 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="link_reached">
+			<param index="0" name="details" type="Dictionary" />
+			<description>
+				Notifies when a navigation link has been reached.
+				The details dictionary may contain the following keys depending on the value of [member path_metadata_flags]:
+				- location: The start location of the link that was reached.
+				- type: Always [code]NavigationPathQueryResult2D.PATH_SEGMENT_TYPE_LINK[/code].
```

Comment:
```suggestion
				- type: Always [constant NavigationPathQueryResult2D.PATH_SEGMENT_TYPE_LINK].
```

Reply:
Maybe also make the keys use `[code]` formatting in each of the descriptions of dictionary keys, so it's clear what they are (and so translators don't think they're English words and translate them).

Reply:
Fixed.